Add français locales

Cryptobunkers opened the pull request.

Cryptobunkers committed:
Create collection.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Create home.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Create category.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update explore.json

Cryptobunkers committed:
Create explore.json

Cryptobunkers committed:
Delete explore.json

Cryptobunkers committed:
Create explore.json

Cryptobunkers committed:
Create filter_sort.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Create profile.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Create create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Create publication.json

Cryptobunkers committed:
Create verification.json

Cryptobunkers committed:
Update verification.json

Cryptobunkers committed:
Update verification.json

Cryptobunkers committed:
Create common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update explore.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update publication.json

Cryptobunkers committed:
Update verification.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Create token_detail.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Create transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Create loyalty.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update loyalty.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update loyalty.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update publication.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update loyalty.json

Cryptobunkers committed:
Update loyalty.json

ahnafalfariza commented on fr/common.json:
```diff
@@ -0,0 +1,384 @@
+		"notification_raffle_type_drop_not_bronze_and_active": "<semibold>Votre niveau d'adhésion a diminué !</semibold>",
+		"notification_raffle_type_drop_not_bronze_and_active_detail": "Vous serez automatiquement inscrit au tirage au sort {{currRaffleType}}",
+		"notification_raffle_type_drop_bronze_and_deactive": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
+		"notification_raffle_type_drop_bronze_and_deactive_detail": "Vous êtes actuellement membre {{currRaffleType}}. Recommencez à <Link>bloquer vos $PARAS</Link> pour les conserver à {{prevRaffleType}}"",
+		"notification_raffle_type_drop_not_bronze_and_deactive": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
```
There is two double quote here causing json error

ahnafalfariza reviewed (commented):
Hi @Cryptobunkers, thanks for the contribution!
Anyway, there is a format issue.

Cryptobunkers committed:
Update common.json

Cryptobunkers commented:
Hi Ahnaf Alfariza,

The error is corrected

Le lun. 30 janv. 2023 à 06:28, Ahnaf Alfariza ***@***.***> a
écrit :

> ***@***.**** commented on this pull request.
>
> Hi @Cryptobunkers <https://github.com/Cryptobunkers>, thanks for the
> contribution!
> Anyway, there is a format issue.
> ------------------------------
>
> In fr/common.json
> <https://github.com/ParasHQ/paras-marketplace-locales/pull/2#discussion_r1090186457>
> :
>
> > +		"notification_raffle_type_drop_bronze_and_deactive_detail": "Vous êtes actuellement membre {{currRaffleType}}. Recommencez à <Link>bloquer vos $PARAS</Link> pour les conserver à {{prevRaffleType}}"",
>
> +		"notification_raffle_type_drop_not_bronze_and_deactive": "<semibold>Votre niveau est tombé à Bronze !</semibold>",
>
>
> There is two double quote here causing json error
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ParasHQ/paras-marketplace-locales/pull/2#pullrequestreview-1274498651>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5OA7VZWWXTSUEUFQGXALY3WU5GQTANCNFSM6AAAAAAUGXNVKM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ahnafalfariza commented:
Thanks!
Here is the deployed version of paras website in french (https://staging-new.paras.id/fr)
Please update if there are translations that are wrong or not quite a right fit

In the meantime, we will check also on our side
cc: @ekkirinaldi

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers commented:
I have updated some translations to fit better. Can you upload files I
updated today ?

Le lun. 30 janv. 2023 à 08:51, Ahnaf Alfariza ***@***.***> a
écrit :

> Thanks!
> Here is the deployed version of paras website in french (
> https://staging-new.paras.id/fr)
> Please update if there are translations that are wrong or not quite a
> right fit
>
> In the meantime, we will check also on our side
> cc: @ekkirinaldi <https://github.com/ekkirinaldi>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ParasHQ/paras-marketplace-locales/pull/2#issuecomment-1408135780>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5OA7VZV6XQFE5HNQMJLPW3WU5XJLANCNFSM6AAAAAAUGXNVKM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update publication.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update explore.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Create token.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update loyalty.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update publication.json

Cryptobunkers committed:
Update token.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update home.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update transaction_modal.json

ahnafalfariza commented:
Hi @Cryptobunkers, could you please fix this error, mainly about the missing coma `,` at the end of the line

<img width="518" alt="image" src="https://user-images.githubusercontent.com/33027382/217475869-2eda3aff-d742-4cfc-820d-e8119fe0862c.png">

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers committed:
Update category.json

Cryptobunkers committed:
Update collection.json

Cryptobunkers committed:
Update common.json

Cryptobunkers committed:
Update create.json

Cryptobunkers committed:
Update filter_sort.json

Cryptobunkers committed:
Update profile.json

Cryptobunkers committed:
Update token_detail.json

Cryptobunkers committed:
Update transaction_modal.json

Cryptobunkers commented:
Hi, it's done.
I have understood why the comma were missing 👌


Le mer. 8 févr. 2023 à 09:31, Ahnaf Alfariza ***@***.***> a
écrit :

> Hi @Cryptobunkers <https://github.com/Cryptobunkers>, could you please
> fix this error, mainly about the missing coma , at the end of the line
>
> [image: image]
> <https://user-images.githubusercontent.com/33027382/217475869-2eda3aff-d742-4cfc-820d-e8119fe0862c.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ParasHQ/paras-marketplace-locales/pull/2#issuecomment-1422219891>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5OA7V4IMPDW2ZL6X2DCMJDWWNKWXANCNFSM6AAAAAAUGXNVKM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
